examples/nextjs-oauth/README.md

```diff
@@ -45,13 +48,13 @@ Now you can start working directly with the sample app:
 4. Create a `.env` file in the root folder, and set the following variables:
 
 ```
-clientID={YOUR_CLIENT_ID)
+clientId={YOUR_CLIENT_ID)
```

Comment:
@janza shouldnt these be 
```
MIRO_CLIENT_ID:""
MIRO_CLIENT_SECRET: ""
MIRO_REDIRECT_URL: ""
MIRO_BOARD_ID: ""
```

Reply:
Good one! In case, I'll apply this edit globally.
@janza enlighten us! 💡

Reply:
This is an older examples that doesn't use our client library. 
So this is correct as is. 

See: https://github.com/miroapp/app-examples/blob/19391811647bea1985b8b6a6a6a13a3e0a50d943/examples/nextjs-oauth/pages/api/authenticate.js#L20